scripts/linters/pylint_extensions_test.py

```diff
@@ -4530,4 +4553,4 @@ def test_non_import_of_text_from_typing_does_not_raise_error(self):
             """)
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_importfrom(
-                node)
+                node)
```

Comment:
Add newline at the end of file (here and elsewhere)

Reply:
Done